samples/gibbs_ensemble/create_fits.py

```diff
@@ -0,0 +1,197 @@
+"""
+Take in all simulation data, compute equilibrium values for every datafile and take means for simulations with same temperature;
+Plot the phase diagram and the fitted critical point.
+"""
+
+import numpy as np
+from scipy.optimize import curve_fit
+from scipy import signal
+import pickle
+import gzip
+import matplotlib.pyplot as plt
+import argparse
+
+parser = argparse.ArgumentParser(
+    description="Computes mean values for every file given as input, computes the mean values for same temperatures and different seeds, tries to compute the critical point using rectilinear and scaling law and plots the phase diagram.")
+parser.add_argument(
+    'files',
+    nargs='+',
+    help="(All) files of the simulated system")
+args = parser.parse_args()
+
+
+# Warmup length to skip at beginning
+WARMUP_LENGTH = int(3e5)
+
+# Lists for (unsorted) densities and temperature
+dens_liquid = []
+dens_gas = []
+errors_liquid = []
+errors_gas = []
+kTs = []
+
+# 3D critical exponent (See Frenkel, Smits: "Understanding Molecular
+# simulation", p.217
+beta = 0.32
```

Comment:
if this is a constant
```suggestion
BETA = 0.32
```